Doc/c-api/contextvars.rst

```diff
@@ -139,11 +139,8 @@ Context object management functions:
    exception will be printed as an unraisable exception using
    :c:func:`PyErr_FormatUnraisable`. Otherwise it should return ``0``.
 
-   There may already be a pending exception set on entry to the callback. In
-   this case, the callback should return ``0`` with the same exception still
-   set. This means the callback may not call any other API that can set an
-   exception unless it saves and clears the exception state first, and restores
-   it before returning.
+   Any pending exception is cleared before the callback is called and restored
```

Comment:
I'd say this should refer to callbacks? (Since you don't call pending exceptions left behind by previous callbacks returning 0.)

Reply:
I'm not sure what you mean—this says "before the callback is called" so it does refer to callbacks.

I pushed the change-return-type-to-void commit to this PR.  Hopefully that resolves the issue.

Reply:
Someone else will have to review this. I am out of time, sorry.